WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginMagicLinkRequestFragment.java

```diff
@@ -362,7 +362,7 @@ public void onAuthEmailSent(OnAuthEmailSent event) {
             if (mForceRequestAtStart) {
                 FragmentManager fragmentManager = getFragmentManager();
                 if (fragmentManager != null) {
-                    fragmentManager.popBackStackImmediate();
+                    fragmentManager.popBackStack();
```

Comment:
This change fixes a race-condition, which would crash the app if there is no screen-transition animation between the "magic link request" screen and "magic link sent" screen.